management/client.go

```diff
@@ -240,6 +277,28 @@ func (m *ClientManager) Delete(id string, opts ...RequestOption) error {
 	return m.Request("DELETE", m.URI("clients", id), nil, opts...)
 }
 
+// CreateCredential creates an client application's client credential.
+func (m *ClientManager) CreateCredential(id string, credential *Credential, opts ...RequestOption) error {
+	return m.Request("POST", m.URI("clients", id, "credentials"), credential, opts...)
+}
+
+// ListCredentials lists all client credentials associated with the client application.
+func (m *ClientManager) ListCredentials(id string, opts ...RequestOption) (c []*Credential, err error) {
```

Comment:
Should we use `clientID` here and in `CreateCredential()` to make it clearer what it is (these methods are not doing clients CRUD), and for consistency with `GetCredential()` and `DeleteCredential()`.

Reply:
Yeah I wasn't sure what naming was best here as we seemed to have a current pattern of defaulting to `id` and then using `xID` and `yID` when there are multiple. But I agree that it's best to ensure it's obvious so I'll move to `clientID` everywhere

Reply:
Fixed in ddf5e4081a7269917695a52062c48c69103241b6